crates/polars-plan/src/dsl/string.rs

```diff
@@ -637,4 +638,10 @@ impl StringNameSpace {
             Some(Default::default()),
         )
     }
+
+    /// Split the string by a substring regex. The resulting dtype is `List<String>`.
+    pub fn flarion_split(self, by: Expr, n: Expr) -> Expr {
```

Comment:
Maybe rename to `flarion_split_by_regex` for more clarity?

Reply:
The idea is to recreate the split() function from Spark, whenever we create a Spark function in polars we use the name flarion_<spark_function_name>, like in flarion_slice

Reply:
Got it 👌